composer/models/huggingface.py

```diff
@@ -180,6 +183,7 @@ def load_huggingface_tokenizer_from_saved_state(
                         s.load_from_serialized_proto(saved_content['content'])
                         with open(tokenizer_file_path, 'wb') as _tmp_file:
                             _tmp_file.write(s.serialized_model_proto())
+
                 hf_tokenizer = transformers.AutoTokenizer.from_pretrained(_tmp_dir, trust_remote_code=True)
```

Comment:
the trust remote code here was also supposed to parameterized in that same pr (https://github.com/mosaicml/composer/pull/2279/files), and somehow got deleted as well. Could you fix that here please?

Reply:
done